boards/arm/ubx_bmd300eval_nrf52832/doc/index.rst

```diff
@@ -0,0 +1,399 @@
+.. _ubx_bmd300eval_nrf52832:
+
+u-blox EVK-BMD-30/35: BMD-300-EVAL, BMD-301-EVAL, and BMD-350-EVAL
+##################################################################
+
+Overview
+********
+
+The BMD-300-EVAL, BMD-301-EVAL, and BMD-350-EVAL hardware provides
+support for the u-blox BMD-300, BMD-301, and BMD-350 Bluetooth 5
+modules, based on the Nordic Semiconductor nRF52832 ARM Cortex-M4F
+CPU and the following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* RADIO (Bluetooth Low Energy)
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/BMD-30-33-35-36-EVAL.png
+	:width: 593px
+	:align: center
+	:alt: BMD-300-EVAL
+
+	BMD-300-EVAL (Credit: u-blox AG)
+
+.. note::
+	The BMD-300-EVAL, BMD-301-EVAL, and BMD-350-EVAL share the same
+	pin headers and assignments. The boards are different only in
+	the module used on the board. The BMD-300 and BMD-301 modules
+	share the same footprint and differ only in the antenna. The
+	BMD-350 module utilizes the nRF52832 CIAA WLCSP package.
+
+More information about the BMD-300-EVAL, BMD-301-EVAL, and
+BMD-350-EVAL and the respective modules can be found at the
+`u-blox website`_. All of the Nordic Semiconductor examples for
+the nRF52 DK (nrf52dk_nrf52810) may be used without modification.
```

Comment:
Wasn't this supposed to be nrf52dk_nrf52832?

Reply:
Fixed - thanks. I'll have nRF52810 and nRF52811 board versions to follow.